components/camera/videosource/webcam_test.go

```diff
@@ -34,7 +34,8 @@ func testGetDrivers() []driver.Driver {
 	}
 	withProps := newFakeDriver("some label", []prop.Media{props})
 	withoutProps := newFakeDriver("another label", []prop.Media{})
-	return []driver.Driver{withProps, withoutProps}
+	isISP := newFakeDriver("some-isp-label", []prop.Media{props})
```

Comment:
_Maybe_ you could hack around this by trying to read an image from each driver but there's a 5 second timeout we'd probably hit for _each_ ISP device. This would also avoid showing a valid camera if there's a Huffman issue. It's also pretty tricky to figure if we can't get an image because the camera is an ISP that hasn't been configured or if it's a valid camera that's in use by another camera component.

Reply:
As I understand this is more of a temporary fix to eliminate confusion from users until we decide if/when to make the changes in those repos. But yeah it definitely doesn't eliminate anything that's videoN, but hopefully only those and a USB webcam as options in the drop down is less confusing than other long isp strings.